test/functional/api/vim_spec.lua

```diff
@@ -1036,5 +1036,25 @@ describe('api', function()
       eq(expected, nvim("list_uis"))
     end)
   end)
-
+  describe('nvim_get_keyboard_mods_state', function()
```

Comment:
BTW if you readd newline here you will have a PR with only added lines in PR statistics and no removed ones.

Reply:
If it looks tidy, I add a new line, otherwise there should be no problem of removing a line as it don't matter of PR stats(is it? As I think work is what is needed)

Reply:
It would just make PR rather unique: I do not remember any PRs which do touch `*.c` files but do not have any removed lines in PR statistics.

Reply:
OK, then its good now